### ISSUE TITLE
Stardoc is failing with Bazel@HEAD in Downstream

### ISSUE THREAD

**Issue by sgowroji** [bazelbuild/stardoc#180]
https://buildkite.com/bazel/bazel-at-head-plus-downstream/builds/3228#0189dda4-c87b-4733-8913-4b74df699726

Platform : Ubuntu, Windows, MacOS

Logs : 
```
ERROR: Traceback (most recent call last):
	File "/private/var/tmp/_bazel_buildkite/9b3c3dc0bf5200f6917a71c964bc4825/external/rules_jvm_external~4.5/private/rules/jetifier.bzl", line 31, column 32, in <toplevel>
		"_jetifier": attr.label(
Error in label: 'cfg = "host"' is deprecated and should no longer be used. Please use 'cfg = "exec"' instead.
ERROR: Analysis of target '//:docs' failed; build aborted: Error loading '@rules_jvm_external~4.5//:extensions.bzl' for module extensions, requested by @@stardoc~override//:MODULE.bazel:22:22: 
```

Steps : 
```
git clone https://github.com/bazelbuild/stardoc.git && cd  stardoc/test/bzlmod
git reset 9800c40d503fff20ee31c5cbfca3b2a6fe7c7ed1 --hard
export USE_BAZEL_VERSION=dbd9e1889f899fe20c7e1195bbde57a8c40b4327
bazel buiild incompatible_disable_starlark_host_transitions --enable_bzlmod  //...
```
https://github.com/bazelbuild/stardoc/issues/145 still failing in downstream

CC Greenteam @comius

**Comment by tetromino** [comment]
Failure at 9800c40d503fff20ee31c5cbfca3b2a6fe7c7ed1 is expected

The fix was in the next commit: a0aa79a09df2d927ad42bb712c8a97586b1d01fa

I'm wondering why the the pipeline using Stardoc at 9800c40d503fff20ee31c5cbfca3b2a6fe7c7ed1? Because it was a tagged release?

**Comment by tetromino** [comment]
Should be fixed by the 0.6.2 release
